src/components/Filter/FilterItem.tsx

```diff
@@ -0,0 +1,56 @@
+﻿import React from 'react';
+import FilterCheckbox from '../FilterCheckbox/FilterCheckbox';
+import { FilterGroup, InputProps, JobType } from '../../types/types';
+import { useFilters } from '../../context/FiltersContext';
+
+interface FilterItemProps {
+   title: string;
+   inputs: InputProps[];
+}
+
+const FilterItem = ({ inputs, title }: FilterItemProps) => {
+   const { filters, setFilters } = useFilters();
+
+   const inputType = {
+      checkbox: ({ name, label, checked, group, type }: InputProps) => (
+         <FilterCheckbox
+            name={name}
+            label={label}
+            checked={checked}
+            group={group}
+            type={type}
+            onChange={changeHandler}
+            isFilterExist={isFilterExist}
```

Comment:
**nitpick:** This flag should not be here - it does not mean anything related to checkbox